src/ngspicetoModelica/NgspicetoModelica.py

```diff
@@ -33,75 +30,79 @@ def readNetlist(self,filename):
                 f = open(filename)
             except Exception as e:
                 print("Error in opening file")
-                print(str(e))
+                print((str(e)))
```

Comment:
use single bracket

Reply:
Resolved under the documentation PR.

Should, I squash my commits, before you merge them for the documentation PR?

Reply:
no, push a new commit in the documentation branch.